README.md

```diff
@@ -1,2 +1,176 @@
+
+> Go to see the live Demo [here!](https://luiscarlosvd.github.io/capstone-project-cookiemasters/) 🙌
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Web Browser (Google Chrome, Microsoft Edge, etc).
+2. Code Editor (Visual Studio Code or similar).
+3. Git.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+
+```sh
+  cd my-desired-folder
+
+  git clone https://github.com/Luiscarlosvd/capstone-project-cookiemasters.git
+
+  cd capstone-project-cookiemasters
+```
+
+### Install
+
+Install this project with:
+
+You have to install linters locally and here is a [documentation](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to do it, just check it out and follow step by step the instrutions.
+
+### Usage
+
+To run the project, execute the following command:
+
+Open *index.html* using live server extension in Visual Studio Code.
+
+### Run tests
+
+> Coming Soon...
+
+### Deployment
+
+> Coming Soon...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 Luis Valera
+
+- GitHub: [@Luiscarlosvd](https://github.com/Luiscarlosvd)
+- Twitter: [@Luiscarlosvd29](https://twitter.com/Luiscarlosvd29)
+- LinkedIn: [Luis Valera](https://www.linkedin.com/in/luis-valera-6a5749267/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add Program Page. 😁
+- [ ] Add JavaScript functionalities. 💪
+- [ ] Add nice backgrounds and more styling. 👏
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like Cookie Master's Project just give me a star! ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse Team, my friends, my family and my girlfriend, they are all my support and inspiration. ✨
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+>Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07). <br>
+>[Image of the background in Main Program by vector_corp on Freepik](https://www.freepik.com/free-vector/abstract-dark-low-poly-background_25861887.htm#query=website%20background%20black&position=14&from_view=search&track=ais). <br>
```

Comment:
- [ ] nice work. Kindly move the appraisal of this people to the acknowledgment section.